inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_base.cpp

```diff
@@ -108,6 +108,9 @@ JitConstants KernelBase::MakeFusedOpsJitConstants(const kernel_selector::base_pa
     if (conf.empty())
         return jit;
 
+    if (params.fused_ops.size() == 1 && params.fused_ops[0].GetType() == KernelType::REORDER)
```

Comment:
Isn't condition below enough to skip reorder processing?

Reply:
If reorder is the only  fused_op but proceed to the below, MakeFusedOpsDeclJit is executed and set HAS_FUSED_OPS true, which results in error.